API.md

```diff
@@ -163,6 +163,48 @@ Assert that the [HTMLElement][] or an [HTMLElement][] matching the
 assert.dom('input[type="text"]').isNotRequired();
 ```
 
+### isVisible
+
+Assert that the [HTMLElement][] or an [HTMLElement][] matching the
+`selector` is visible. Visibility is determined with the hueristic
+used in [jQuery's :visible pseudo-selector](https://github.com/jquery/jquery/blob/2d4f53416e5f74fa98e0c1d66b6f3c285a12f0ce/src/css/hiddenVisibleSelectors.js#L12),
+specifically:
+
+-   is the element's offsetWidth non-zero
+-   is the element's offsetHeight non-zero
+-   is the length of an element's DOMRect objects found via getClientRects() non-zero
```

Comment:
just to clarify, this means it's visible on the page, but not necessarily that it's within the currently rendered viewport, correct?

Reply:
Yes, that's exactly what it means.

Reply:
cool, can we add that to the docs so that they read a little easier? 😉

Reply:
Yes. Absolutely.